data/mods/Aftershock/items/magazine/energy_cell.json

```diff
@@ -44,5 +44,21 @@
     "reload_time": 5,
     "flags": [ "NO_UNLOAD" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 600 } } ]
+  },
+  {
+    "id": "afs_archeotech_cartridge",
+    "copy-from": "afs_cartridge",
+    "type": "MAGAZINE",
+    "name": { "str": "chromoelectric cartridge" },
```

Comment:
Someone please find a Quantum Physicist to tell me whether this name makes sense.
The alliteration is nice tho.

Reply:
What does chromo mean?

Reply:
In this context it relates to 
https://www.britannica.com/science/quantum-chromodynamics

Reply:
I'm not sure chromo- (color, hue) would be an appropriate combination there since electricity does not inherently have... well, a color. Although these sometimes psuedo greek/latin word combos get a LOT of leeway, because most people don't know what they mean of course.

Perhaps "varioelectric cartridge"? vario- in this case meaning *different*, *vario*us, etc... Doesn't really click for me as one of those fun multi-layered compound word salads though.

Reply:
Radioelectric cartridge, since it's using radioactivity?